README.md

```diff
@@ -0,0 +1,68 @@
+# Orion
+The Ultimate Torrent Client and Streamer for Linux Desktop
+
+<p align="center">
+  <img src="https://github.com/keshavbhatt/orion-issue-tracker/blob/master/images/banner_aVrjYhn.jpeg?raw=true">
+</p>
+
+Orion is a powerful, light-weight, fast BitTorrent client with beautiful user experience and fastest Media streaming capabilities.
+
+To achieve all user comfort, Orion comes with an in-built powerful media player backend which very comfortably integrates with Orion to serve the best user experience while streaming media files over the BitTorrent protocol. 
+
+<br/>
+
+## Key features of Orion:
+
+**Key features of Orion Torrent Client & Streamer:**
+- Simple and easy to use feature rich torrent client
+- Ultra-fast downloading algorithms optimized for seamless streaming
+- Stream torrents containing video, audio & images
+- Speed throttling to limit download and upload speed per task
+- Share files and folders  with users by hosting torrent server (peer to peer file/folder sharing)
+- Built in media player to handle streaming needs
+- Super-efficient peer selection and choking
+- Multi-threaded download manager
+- Creates Streaming server which can be accessed from other services
+- Peers discovery via Tracker servers, DHT (Distributed Hash Table), Local Peer Discovery (LSD), Peer Exchange (PEX), Web seed networks, web torrent protocol using webRTC .
+- Create torrent, share hash to your friend to share files
+- Opens magnet links, Torrent files & torrent hashes.
+- App lock (no one can access app without set pass-code)
+- Supports plugins
+        - Torrent host server(create and share files by hosting a torrent)
+        - Torrent search engine plugin
+        - Discover movie browser plugin
+        - YouTube player & downloader plugin
+        - Torrent meta plugin (shows quick overview of selected torrent)
+        - History manager plugin (keeps your opened torrents safe for future)
+
+<br/>
+
+## Install Orion on Linux Desktop
```

Comment:
this README will NEVER be  seen by users.

Reply:
Is it okay to keep? :confused: